pkg/install/generate.go

```diff
@@ -103,7 +104,7 @@ func GenerateInstallerJob(
 			ValueFrom: &corev1.EnvVarSource{
 				SecretKeyRef: &corev1.SecretKeySelector{
 					LocalObjectReference: cd.Spec.PullSecret,
-					Key:                  ".dockercfg",
+					Key:                  dockerConfigSecretKey,
```

Comment:
extra credit: use the constant from corev1

Reply:
On it.